circuit/package.json

```diff
@@ -24,7 +24,7 @@
     "crypto"
   ],
   "dependencies": {
-    "@axiom-crypto/core": "2.3.7",
+    "@axiom-crypto/core": "2.3.8-alpha.0",
```

Comment:
Should this be an alpha?

Reply:
@yi-sun this is correct; 2.3.8-alpha.0 removes the chainId checks in core. it's an alpha release because it we had to do a high priority v2.0.6 client release in the middle of the refactor. anyone who tries to download core right now will by default get the v2.3.7 full release since it's tagged as `latest` still